plugins/systemair/__init__.py

```diff
@@ -53,6 +55,7 @@ def init_serial_connection(self, serialport, slave_address):
             return False
 
     def _read_modbus(self):
+        self._lockmb.acquire()
```

Comment:
In case of connection error (see `init_serial_connection()` below) the lock will never be release. I don't know if this is a real problem since a connection problem should usually not occur, but maybe this plugin tries to recover by re-trying to connect in the next `_read_modbus()` call which then can't acquire the lock.